public/components/common/hooks/use-value-suggestion.ts

```diff
@@ -31,6 +31,7 @@ export interface IValueSuggestion {
 
 export const useValueSuggestion = (
   filterField: string,
+  filterDrillDownValue,
```

Comment:
I am not sure about this name inside the `useValueSuggestion` hook. A hook should concern itself with it's own behavior and not where it is used, it is not a bad idea to accept filters as a parameter but I feel that naming it `DrillDownValue` ties it too closely with the behavior of other components. Maybe replace it by `boolFilters` internally.

This parameter should also have a type. I would suggest
```
interface BoolFilter {
    field:string;
    value:string;
}
export const useValueSuggestion = (
    boolFilters : BoolFilter[] = [],
    ...
```
This snippet of code also adds a default value to this parameter so you can always count on having an array here.